lesson02/assignment/css/_mixin.scss

```diff
@@ -0,0 +1,8 @@
+/*mixins for media queries*/
+@mixin for-phone-only {
+    @media (max-width: 500px) { @content; }
+}
+
+@mixin for-smaller-phone {
+    @media (max-width: 375px) { @content; }
```

Comment:
These are excellently clear mixin names! Very easy to understand and use. However, industry best practices are for naming conventions to be more abstract, as sizes don't always correlate with particular devices. I know names like "extra-small" and "large" aren't nearly so evocative, but they are more accurate. Maybe there's a middle path? "small-device", "extra-small-screen", something else?